tools/benchmark/src/mocha/customOutputRunner.ts

```diff
@@ -0,0 +1,101 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Test } from "mocha";
+
+import type { BenchmarkDescription, MochaExclusiveOptions } from "../Configuration";
+import { timer } from "../timer";
+
+/**
+ * Options to configure a benchmark that reports custom measurements.
+ *
+ * @internal
```

Comment:
It seems like all your new stuff is marked `@internal`. Personally I don't think that makes a lot of sense. We document that "There are no API stability guarantees for packages in the @fluid-experimental and @fluid-internal scopes." (source: https://fluidframework.com/docs/build/releases-and-apitags/ )

Since this package is an independently versioned and published package, all consumers (including within this repo) consume it via a "^" and use its public API. If you add internal APIs, there are no places where we could safely use those unless we switched to pinning an exact version. Also since this is a dev tool, we can make breaking API changes and bump the version accordingly anyway without any major downsides, so there is no reason to mess with separate internal APIs.

Reply:
Small nit: benchmark is in scope `@fluid-tools`, not `@fluid-experimental` or `@fluid-internal`. I don't know if we have explicit documentation about what we do/don't guarantee for that scope.

I agree with `@internal` being unnecessary though.